webapp/parsing_csv.py

```diff
@@ -7,6 +7,7 @@
                            CREDIT_FOR, DEBIY_FOR)
 
 
+
 class Error_csv_file(Exception):
```

Comment:
В PEP8 принято специальное соглашение, как называть эксепшены.
```suggestion
class CsvFileError(Exception):
```

Reply:
поправил